aws/resource_aws_network_interface.go

```diff
@@ -323,17 +323,21 @@ func resourceAwsNetworkInterfaceUpdate(d *schema.ResourceData, meta interface{})
 		d.SetPartial("private_ips")
 	}
 
-	request := &ec2.ModifyNetworkInterfaceAttributeInput{
-		NetworkInterfaceId: aws.String(d.Id()),
-		SourceDestCheck:    &ec2.AttributeBooleanValue{Value: aws.Bool(d.Get("source_dest_check").(bool))},
-	}
+	// ModifyNetworkInterfaceAttribute needs to be called after creating an ENI
+	// since CreateNetworkInterface doesn't take SourceDeskCheck paramter.
```

Comment:
Fix for linting error:

```suggestion
	// since CreateNetworkInterface doesn't take SourceDeskCheck parameter.
```